PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
+  var tableView: UITableView!
+  
+  var rowHeight: CGFloat!
```

Comment:
Again, it's better to make non-optional and use default value
```suggestion
  var rowHeight: CGFloat = 100
```
Also consider access levels for properties, I think most of them should be `private`

Reply:
Done